dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -91,4 +96,18 @@ public Entry updateAliases(@ApiParam(hidden = true) @Auth User user,
         return c;
     }
 
+    @GET
+    @Path("/{id}/collections")
+    @Timed
+    @UnitOfWork
+    @ApiOperation(value = "Get a published entry's collections and organizations it's part of", notes = "Entry must be published", response = CollectionOrganization.class, responseContainer = "List")
```

Comment:
Supposedly, `A preposition is a bad thing to end a sentence with`. :) But I don't feel that strongly about it.